package.json

```diff
@@ -1,4 +1,94 @@
 {
   "name": "builder-react-component",
-  "version": "0.0.0"
+  "version": "0.0.1",
+  "description": "Builder Archetype - React Component",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FormidableLabs/builder-react-component.git"
+  },
+  "author": "Ryan Roemer <ryan.roemer@formidablelabs.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/FormidableLabs/builder-react-component/issues"
+  },
+  "homepage": "https://github.com/FormidableLabs/builder-react-component",
+  "scripts": {
+    "npm:postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || builder run build-lib",
```

Comment:
Something I've been thinking about doing with `bolt` is to actually split these out into a `task.json` so it doesn't clutter up the package's scripts, and since you actually have a script to `install` the dependencies for the `Archetype`, what are your thoughts on maybe having a `builder.json` that houses `tasks`, `dependencies` and `devDependencies`.

I know this kinda stands in a stark contrast to keeping things in the `package.json` because that' where they'd normally live, but I've found that in my cursory exploration of trying this out, it cleans up my `package.json` for the archetype itself significantly.